sanityChecks.groovy

```diff
@@ -1,36 +1,29 @@
-def linux_build() {
-    sh 'echo "works"'
-}
-def windows_build() {
-    bat 'echo \'works\''
-}
-timeout(60) {
-    stage('Verify VMAgents templates') {
-        parallel 'Custom Ubuntu without init script': {
-            node('s-cust-no-init-ubuntu') {
-                linux_build()
-            }
-        },
-        /*'Custom Ubuntu with init script': {
-            node('s-cust-init-ubuntu') {
-                linux_build()
-            }
-        },*/
-        'Reference Ubuntu with init script': {
-            node('s-ref-ubuntu') {
-                linux_build()
-            }
-        },
-        'Custom Windows with init script': {
-            node('s-cust-windows') {
-                windows_build()
-            }
-        },
-        'Reference Windows with init script': {
-            node('s-ref-windows') {
-                windows_build()
+#!groovy
+pipeline {
+    agent none
+    options {
+        timeout(time: 60, unit: 'MINUTES')
+    }
+    stages {
+        stage('Verify VMAgents templates') {
+            steps {
+                 parallel ( failFast: true,
+                    'Custom Ubuntu without init script': {
+                        runSanityChecks('s-cust-no-init-ubuntu')
+                    },
+                    /*'Custom Ubuntu with init script': {
```

Comment:
Why is this commented out? Suggest either removing it or adding a comment describing why it's there (in the file, not on github)